nupic/algorithms/anomaly_likelihood.py

```diff
@@ -166,12 +168,7 @@ def _calcSkipRecords(numIngested, windowSize, learningPeriod):
       'settle down'.
     """
     numShiftedOut = max(0, numIngested - windowSize)
-    return max(0, learningPeriod - numShiftedOut)
-
-
-  def forceModelRefresh(self):
-    """Force refresh of the anomaly likelihood model's state"""
-    self._distribution = None
+    return min(numIngested, max(0, learningPeriod - numShiftedOut))
```

Comment:
I caught this once I implemented unit tests for `_calcSkipRecords`. I think that the user of the value handled it gracefully, but it was potentially error-prone and unintuitive.